src/reflect/scala/reflect/internal/Symbols.scala

```diff
@@ -725,7 +725,7 @@ trait Symbols extends api.Symbols { self: SymbolTable =>
 
     def setFlag(mask: Long): this.type   = { _rawflags |= mask ; this }
     def resetFlag(mask: Long): this.type = { _rawflags &= ~mask ; this }
-    def resetFlags() { rawflags &= TopLevelCreationFlags }
+    def resetFlags() { rawflags = 0 }
```

Comment:
i checked in dotty, there's [`PackageCreationFlags`](https://github.com/lampepfl/dotty/blob/845b98186047f38013e2f6aa35508e974eedafb7/src/dotty/tools/dotc/core/Flags.scala#L504) which is the same as scala's `PackageFlags`, which are in fact `TopLevelCreationFlags`. but dotty doesn't have something like `TopLevelCreationFlags`. dotty actually seems to create a new symbol instead of updating the existing one (https://github.com/lampepfl/dotty/blob/845b98186047f38013e2f6aa35508e974eedafb7/src/dotty/tools/dotc/typer/Namer.scala#L258-L279).